CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## unreleased
 
+## v4.6.0 - 2023.05.02
+
+* Update CSI driver for Kubernetes 1.27
+  [[GH-486]](https://github.com/digitalocean/csi-digitalocean/pull/486)
+
```

Comment:
Would you mind adding a small entry for #487?

Reply:
Sure! Small entry added.